scarlet/source.py

```diff
@@ -617,12 +346,14 @@ def __init__(
         model_frame,
         sky_coord,
         observations,
-        obs_idx=0,
+        coadd=None,
+        bg_cutoff=None,
         thresh=1.0,
         flux_percentiles=None,
         symmetric=False,
         monotonic="flat",
         shifting=False,
+        min_grad = 0.2,
```

Comment:
Please change `min_grad` to `0.0`.

Reply:
Good point! Setting min_grad to 0 by default brought back the loss at 31652.619806746377